nodejs/dist/uws.js

```diff
@@ -318,7 +332,7 @@ class Server extends EventEmitter {
                                 } else {
                                     // todo: send code & message
                                     if (this._lastUpgradeListener) {
```

Comment:
This is my bad, we should always close a socket that matches our path but fails the verification (only we have this path so we need to close it in all cases)